pkgs/by-name/ga/gambit-project/package.nix

```
@@ -0,0 +1,31 @@
+{ stdenv
+, fetchFromGitHub
+, autoreconfHook
+, wxGTK31
+, lib
+, withGui ? true
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gambit-project";
+  version = "16.1.1";
+
+  src = fetchFromGitHub {
+    owner = "gambitproject";
+    repo = "gambit";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-ElPzJDQ1q+i1OyliychSUA9pT6yGSwjn/sKV0JX5wrQ=";
+  };
+
+  nativeBuildInputs = [ autoreconfHook ];
+
+  buildInputs = lib.optional withGui wxGTK31;
```

```suggestion
  nativeBuildInputs = [ 
    autoreconfHook 
  ] ++ lib.optionals withGui [
    wxGTK31
  ];

  buildInputs = lib.optionals withGui [ 
    wxGTK31 
  ];
  
  strictDeps = true;
```

P.S.: is there any specific flag to set in order to build with GUI?